apps/server/src/modules/authentication/services/authentication.service.ts

```diff
@@ -1,22 +1,21 @@
-import { AccountService, Account } from '@modules/account';
+import { CreateJwtPayload } from '@infra/auth-guard';
+import { Account, AccountService } from '@modules/account';
+import type { ServerConfig } from '@modules/server';
 import { Injectable } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
 import { JwtService } from '@nestjs/jwt';
-// invalid import, can produce dependency cycles
-import type { ServerConfig } from '@modules/server';
 import { randomUUID } from 'crypto';
 import jwt, { JwtPayload } from 'jsonwebtoken';
 import { BruteForceError, UnauthorizedLoggableException } from '../errors';
-import { CreateJwtPayload } from '../interface/jwt-payload';
-import { JwtValidationAdapter } from '../helper/jwt-validation.adapter';
-import { LoginDto } from '../uc/dto';
+import { JwtWhitelistAdapter } from '../helper/jwt-whitelist.adapter';
 import { UserAccountDeactivatedLoggableException } from '../loggable/user-account-deactivated-exception';
+import { LoginDto } from '../uc/dto';
 
 @Injectable()
 export class AuthenticationService {
 	constructor(
 		private readonly jwtService: JwtService,
-		private readonly jwtValidationAdapter: JwtValidationAdapter,
+		private readonly jwtValidationAdapter: JwtWhitelistAdapter,
```

Comment:
```suggestion
		private readonly jwtWhitelistAdapter: JwtWhitelistAdapter,
```
Rename `jwtValidationAdapter` to `jwtWhitelistAdapter` to make it more clear what is used here.